internal/pkg/v2/infrastructure/redis/event.go

```diff
@@ -130,6 +130,16 @@ func deleteEvent(conn redis.Conn, id string) (edgeXerr errors.EdgeX) {
 	return edgeXerr
 }
 
+func scrubRecordsByKey(conn redis.Conn, key string) error {
```

Comment:
rename to `deleteRecordsByKey` as it is more explicate than `scrub` and follows the names used for the higher level functions that get us to this one.

Reply:
Renamed